linters/actionlint/test_data/actionlint_v1.6.26_CUSTOM.check.shot

```diff
@@ -1,5 +1,4 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
-// trunk-upgrade-validation:RELEASE
 
 exports[`Testing linter actionlint test CUSTOM 1`] = `
 {
```

Comment:
This change (and similar) is not related to trufflehog, but something @Tyler asked me to do to update the tests.